src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -244,6 +251,8 @@ If you disable or don't configure this policy, no predefined rules are applied.
       <string id="MwbValidateRemoteIp">Validate remote machine IP Address</string>
       <string id="MwbDisableUserDefinedIpMappingRules">Disable user defined IP Address mapping rules</string>
       <string id="MwbPolicyDefinedIpMappingRules">Predefined IP Address mapping rules</string>
+      <string id="AllowPowerToysAdvancedPasteOnlineAIModels">Advanced Paste: Allow using online AI models</string>
```

Comment:
must be a merge conflict. This policy was renamed. I have removed the `Advanced Paste: ` part in a PR.